merkletree/pad.go

```diff
@@ -121,11 +122,13 @@ func (pad *PAD) LatestSTR() *SignedTreeRoot {
 	return pad.latestSTR
 }
 
-func (pad *PAD) TB(name string, value []byte) (*TemporaryBinding, error) {
+func (pad *PAD) Sign(msg ...[]byte) []byte {
+	return pad.signKey.Sign(bytes.Join(msg, nil))
+}
+
+func (pad *PAD) NextIndex(name string) []byte {
```

Comment:
Or we can remove the prefix, just like `pad.Set`? Then we may need to rename `Lookup` to something like `LookupInLatestEpoch`.